api/pending_test_runs.go

```diff
@@ -0,0 +1,33 @@
+// Copyright 2019 The WPT Dashboard Project. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package api
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/web-platform-tests/wpt.fyi/shared"
+)
+
+// apiPendingTestRunsHandler is responsible for emitting JSON for
+// all the pending test runs.
+func apiPendingTestRunsHandler(w http.ResponseWriter, r *http.Request) {
+	ctx := shared.NewAppEngineContext(r)
+	store := shared.NewAppEngineDatastore(ctx, true)
+
+	q := store.NewQuery("PendingTestRun")
+	var runs []shared.PendingTestRun
+	if _, err := store.GetAll(q, &runs); err != nil {
```

Comment:
We need to hold onto the keys so that we can backfill the ID field of the pending runs (and this should be tested).

Reply:
Done in https://github.com/web-platform-tests/wpt.fyi/pull/1432/commits/99c9c375cf8fca11595e683264e81a9b2d809ab1